pkg/splunk/controller/messages.go

```diff
@@ -0,0 +1,5 @@
+package controller
```

Comment:
Is this needed?

Reply:
Yes it is required. Every go file must be a part of a package.

Reply:
NVM, i just noticed its a new file.